priv/perf/apps/load_test/lib/scenario/deposits.ex

```diff
@@ -53,17 +54,24 @@ defmodule LoadTest.Scenario.Deposits do
   end
 
   def create_deposit_and_make_assertions(session) do
+    {_, session} =
+      Metrics.run_with_metrics(
+        fn ->
+          do_create_deposit_and_make_assertions(session)
+        end,
+        "test"
```

Comment:
```suggestion
        "deposit_test"
```

Reply:
I changed it to `test` in `utxo` (`transactions`) as well. Because in test_runner I check if `test_failure` is nil to determine if it was successful.